packages/hdwallet-provider/test/provider.test.ts

```diff
@@ -224,19 +222,19 @@ describe("HD Wallet Provider", function () {
         mnemonic: {
           phrase: mnemonicPhrase
         },
-        url: `http://localhost:${port}`,
+        provider: ganacheProvider
         // polling interval is unspecified
       });
-      assert.ok(provider.engine,
-        "Web3ProviderEngine instantiated");
-      // @ts-ignore - _blockTracker is not officially part of the interface
-      assert.ok(provider.engine._blockTracker,
-        "PollingBlockTracker instantiated");
+      assert.ok(provider.engine, "Web3ProviderEngine instantiated");
+      assert.ok(
+        (provider.engine as any)._blockTracker,
+        "PollingBlockTracker instantiated"
+      );
```

Comment:
formatting hooks wrapped this line, which made the `@ts-ignore` no longer apply to the correct place, so I opted to replace it w/ a cast to `any`, as that's at least enshrined in proper syntax.